DailyReportWeb/Controllers/Api/PolicyController.cs

```diff
@@ -26,9 +28,13 @@ public IEnumerable<PolicySummary> Get()
         }
 
         // GET: api/Policy/5
-        public Policy Get(string id)
+        public ReportSettings Get(long id)
         {
-            return new Policy();
+            using (var db = new ReportsDb())
+            {
+                var baseUrl = User.GetBaseUrl();
+                return db.ReportSettings.SingleOrDefault(qr => qr.ProjectId == id && qr.BaseUrl == baseUrl);
```

Comment:
Aici nu e ok ca intorci SingleOrDefault. Ar trebui Single, in asa fel incat daca nu gasesti nici un element sa intoarca o exceptie.